scout/server/blueprints/cases/controllers.py

```diff
@@ -228,6 +228,19 @@ def case(store, institute_obj, case_obj):
         individual["phenotype_human"] = pheno_map.get(individual["phenotype"])
         case_obj["individual_ids"].append(individual["individual_id"])
 
+        if individual.get("bionano_access") and not individual.get("fshd_loci"):
+            fshd_loci = bionano_access.get_fshd_report(
```

Comment:
This info is only required when you load the case_bionano.html template, but if it's here you trigger a request to bionano every time that somebody opens a case page. At the same time the case_bionano.html template doesn't need all the info required to build the case page.
Why instead don't you create a dedicated controller to just retrieve the necessary info for the bionano template?

Reply:
Sure, I like! The rationale was that only a handful would have "bionano_access" defined anyway.